### ISSUE TITLE
Catalina Nesting?

### ISSUE THREAD

**Issue by james-grissom** [synapsepd/MacOS-OneDrive-KFM#4]
Love this tool, thanks so much!
I'm having an issue where this is causing the symlinks not to work as desired. It's not removing the built-in folder, it's just putting a new "documents" and "desktop" folder under them with the OD4B data. Is this just a Catalina bug that needs investigation? No issues at all on Mojave and below.

**Comment by samspade21** [comment]
It sounds like this was resolved by adding a MDM profile that lets outset have access to the full disk.  A sample profile has been added to the repo.  It is called outset-whitelist.mobileconfig

**Comment by csteelatgburg** [comment]
I'm experiencing the same issue and noticed that the mobileconfig file references python2 in the identifier, but Outset now uses python3. I tried changing the identifier to identifier "com.apple.python3" and anchor apple but that did not resolve the issue. I have also tried using the identifier for /usr/local/outset/python3 which is:
identifier python3 and anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists */ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* exists */ and certificate leaf[subject.OU] = DJ3H93M7VJ

and this also didn't work.
Note, this is on a machine running 10.15.7 with Outset 3.0.3 and Python 3.9.
